www/js/equation.js

```diff
@@ -556,6 +556,28 @@ define([
 
 		setLogging: function(logger){
 			this._logger = logger;
+		},
+
+		/*getVariableStrings returns the constituent variables as strings in an array inside a user entered equation string
+		*/
+		getVariableStrings: function(/*String*/ equation){
+			var eqs = equation.split(this.equalto);
+			if(eqs.length != 2){
+				throw new Error("Wrong number of equal to symbols in the equation");
+			}
+			var expressions = [];
+			try{
+				array.forEach(eqs, function(eq, count){
+					expressions[count] = Parser.parse(eq);
+				}, this);
+			}catch(e){
+				throw e;
```

Comment:
Why a try-catch block if we're only just going to throw the error anyway?

Reply:
Logging the error sounds good to me.

Reply:
Can we push all the logging to a separate task later and leave a note there now that we have more prioritized tasks ?

Reply:
Sure.